pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -173,7 +174,8 @@ func (configgen *ConfigGeneratorImpl) buildClusters(proxy *model.Proxy, req *mod
 	return resources, model.XdsLogDetails{AdditionalInfo: fmt.Sprintf("cached:%v/%v", cacheStats.hits, cacheStats.hits+cacheStats.miss)}
 }
 
-func shouldUseDelta(updates *model.PushRequest) bool {
```

Comment:
This is supposed to be generic and used for all xDS?

Reply:
no, it is only for cds

Reply:
Otherwise it should not be in this file

Reply:
I think I only added this and my intention was to reuse this across all xDS

Reply:
I will revert this  change, do it when necessary